felix/detector/feature_detect.go

```diff
@@ -12,11 +12,13 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package iptables
+package detector
```

Comment:
I think `detector` is a very broad name and `features` or `environment` or something like that might be better. I would prefer the former as then you can have `features.Detector` instead of `detector.FeatureDetector` and `features.Features` (which is a little unfortunate).

Reply:
renamed it to `environment`.